modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -321,7 +321,7 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
       return version;
     } catch (e) {
       // if that fails, and we aren't supporting p2wsh, then we are done and did not find a version
```

Comment:
Update comment to `and we aren't supporting SegWit`?

Reply:
Actually it's about `bech32` - there may be a coin that has segwit support but not bech32 support, in which case we should forgo the parsing attempt

Will fix. Also the address parsing should fully live in utxolib probably.